src/http.js

```diff
@@ -52,9 +53,13 @@ class HttpClient {
     };
     var attr;
     var header;
+    var attachments = []
     var mergeOptions = ['headers'];
+    if (exoptions) {
+      attachments = exoptions.attachments || [];
+    }
 
-    if (typeof data === 'string') {
+    if (typeof data === 'string' && attachments.lemgth) {
```

Comment:
`attachments.lemgth` => `attachments.length === 0`?

Reply:
I did not understand what the correction would be

Reply:
Please note `length` is correct instead of `lemgth`.

Reply:
I assume you are testing if the attachments array is empty.

Reply:
silly mistake, now it goes